mariadb-10.11.yaml

```diff
@@ -148,6 +148,16 @@ pipeline:
       rm -rf "${{targets.destdir}}"/usr/lib/libmariadb.so*
       rm -rf "${{targets.destdir}}"/usr/lib/pkgconfig/libmariadb.pc
 
+      rm -rf "${{targets.destdir}}"/usr/bin/mysql_client_test \
+        "${{targets.destdir}}"/usr/bin/mysql_client_test_embedded \
+        "${{targets.destdir}}"/usr/bin/mariadb-client-test \
+        "${{targets.destdir}}"/usr/bin/mariadb-client-test-embedded \
+        "${{targets.destdir}}"/usr/bin/mariadb-test \
+        "${{targets.destdir}}"/usr/bin/mariadb-test-embedded \
+        "${{targets.destdir}}"/usr/bin/mysqltest \
+        "${{targets.destdir}}"/usr/bin/mysqltest_embedded \
+        "${{targets.destdir}}"/usr/mysql-test
```

Comment:
this is the path that is deleted that contained the "bad" /usr/mysql-test/plugin/connect/connect/std_data/JdbcMariaDB.jar

Reply:
Great!